prequest.js

```diff
@@ -2,8 +2,28 @@
 
 var Promise = require('bluebird');
 var request = require('request');
+var util = require('util');
 var objectAssign = require('object-assign');
 
+function atMost(obj, max) {
```

Comment:
No need to shorten the body.
I think it tends to be short and concise (as picked up by the tests).